README.md

```diff
@@ -57,8 +57,6 @@ The SpeedUpAmerica project utilizes the following technologies for operation:
 
 These instructions work on Linux, Windows and MacOS and only need to be performed once, unless you reset your database or config files.
```

Comment:
Can we mention that this is for Win Pro not Win Home?

Reply:
The PR has been merged, but I do agree that we should clearly state Pro as Docker Toolkit is an undesirable solution.